src/Core/Auth/Models/Business/Tokenables/OrgUserInviteTokenable.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Text.Json.Serialization;
+using Bit.Core.Entities;
+using Bit.Core.Tokens;
+
+namespace Bit.Core.Auth.Models.Business.Tokenables;
+
+public class OrgUserInviteTokenable : ExpiringTokenable
+{
+    // TODO: Ideally this would be internal and only visible to the test project.
```

Comment:
question: Can we add a tech debt ticket to go along with this `TODO`?

Reply:
After discussion with @justindbaur , I've added a platform tech debt item here: https://bitwarden.atlassian.net/browse/PM-4317 and in code.